fix(spanner): DeleteSampleData sample

laljikanjareeya opened the pull request:
[PR#1111](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1111) failing [TestDeleteSampleData](https://source.cloud.google.com/results/invocations/873b1a6a-39fd-4d49-b254-817029ee8a3a/targets/github%2Fdotnet-docs-samples%2Fspanner%2Fapi%2FSpannerTest%2FTestResults/tests) Test as in sample was not added `await` while deleting the tables.
It's giving below error:
`Error: Google.Cloud.Spanner.Data.SpannerException : Cannot drop table UpcomingSingers with interleaved tables: UpcomingAlbums.`

laljikanjareeya commented:
@googlebot I fixed it.

amanda-tarafa commented on spanner/api/Spanner/Program.cs:
```diff
@@ -2016,11 +2016,11 @@ AlbumTitle   STRING(MAX)
                 return 0;
             }
-            async void DeleteTable(SpannerConnection connection, string table)
+            async Task DeleteTable(SpannerConnection connection, string table)
```
nit: rename to DeleteTableAsync

laljikanjareeya commented:
I fixed it.

*Thanks & Regards*
*Lalji Kanjareeya*


On Wed, Jul 1, 2020 at 5:01 PM googlebot <notifications@github.com> wrote:

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://cla.developers.google.com/> to double check)? If these were
> authored by someone else, then they will need to sign a CLA as well, and
> confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fdotnet-docs-samples%2Fpull%2F1114>
> for more info*.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1114#issuecomment-652364420>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALBOLJBFUJFBT3O7KFQYYF3RZMNC5ANCNFSM4OIKCK4Q>
> .
>

skuruppu commented:
@amanda-tarafa the last test run has an unrelated [failure](https://source.cloud.google.com/results/invocations/5cd50161-0286-4fe9-9a72-5bf063d2b0f6/targets). So we'll need your help to force-merge this once the new set of runs finish assuming there are no spanner test failures.
